controllers/api_controller_integration_test.go

```diff
@@ -174,7 +174,7 @@ var _ = Describe("APIRule Controller", Serial, func() {
 									Route(builders.RouteDestination().Host(testOathkeeperSvcURL).Port(testOathkeeperPort)).
 									Headers(builders.NewHttpRouteHeadersBuilder().SetHostHeader(serviceHost).Get()).
 									CorsPolicy(defaultCorsPolicy).
-									Timeout(time.Second * helpers.DEFAULT_HTTP_TIMEOUT))
+									Timeout(time.Second * 180))
```

Comment:
This `180` value is shared across every test, maybe we should have it as constant at least here?

Reply:
Good idea, just did a find replace and didn't check that it can be an constant in the test.